stacker/providers/aws/default.py

```diff
@@ -207,18 +208,46 @@ def destroy_stack(self, stack, **kwargs):
         return True
 
     def create_stack(self, fqn, template_url, parameters, tags, **kwargs):
-        logger.debug("Stack %s not found, creating.", fqn)
-        logger.debug("Using parameters: %s", parameters)
-        logger.debug("Using tags: %s", tags)
-        retry_on_throttling(
-            self.cloudformation.create_stack,
-            kwargs=dict(StackName=fqn,
-                        TemplateURL=template_url,
-                        Parameters=parameters,
-                        Tags=tags,
-                        Capabilities=["CAPABILITY_NAMED_IAM"]),
-        )
-        return True
+        try:
+            logger.debug("Stack %s not found, creating.", fqn)
+            logger.debug("Using parameters: %s", parameters)
+            logger.debug("Using tags: %s", tags)
+            logger.debug("Using template_url: %s", template_url)
+            retry_on_throttling(
+                self.cloudformation.create_stack,
+                kwargs=dict(StackName=fqn,
+                            TemplateURL=template_url,
+                            Parameters=parameters,
+                            Tags=tags,
+                            Capabilities=["CAPABILITY_NAMED_IAM"]),
+            )
+            return True
+        except botocore.exceptions.ClientError as e:
+            if e.response['Error']['Message'].startswith('TemplateURL'):
```

Comment:
One concern I have about this is that this regression isn't the only way you could end up with an error from CF that starts with `TemplateURL`.  Some others I can think of just based on the full error message `TemplateURL must reference a valid S3 object to which you have access`:

- The templateURL points to an invalid object (not sure what that means, I think it might be a separate error if the object just doesn't exist)
- They might not have access to an existing object

The other issue is fi there are other potential errors that start with `TemplateURL`.

Not sure what the best way is to make this more robust.  Any thoughts?